tools/dgeni/templates/method.template.html

```diff
@@ -50,7 +50,7 @@
     </td>
     <td class="docs-api-method-parameter-description-cell">
       <p class="docs-api-method-parameter-description">
-        {$ parameter.content | marked | safe $}
+        {$ parameter.content or parameter.description | marked | safe $}
```

Comment:
I think this is something that was added as a trick when `.description` did not work for inherited members. I fixed this a couple of weeks ago so we should be able to rely on `.content` everywhere I think.

Reply:
I checked it on the latest master. I generated the api doc for checkbox/testing. The MethodMemberDoc contains the information in the description fiel
![dgeni-description](https://user-images.githubusercontent.com/33690815/143475261-34c43397-23d2-4a09-a65b-670422ee13de.png)
d.

Reply:
Ops. sorry. I meant to write we should only rely on `description` and not have the fallback here.

Reply:
```suggestion
        {$ parameter.description | marked | safe $}
```